samples/RemoteAuth/OIDC/OIDCAuth/Global.asax.cs

```diff
@@ -17,7 +17,8 @@ protected void Application_Start()
             SystemWebAdapterConfiguration.AddSystemWebAdapters(this)
                 .AddProxySupport(options => options.UseForwardedHeaders = true)
                 .AddRemoteAppServer(remote => remote
-                    .Configure(options => options.ApiKey = "test-key")
+                    // Do not re-use this ApiKey; every solution should use a unique ApiKey
```

Comment:
Suggestion: make this come from config or something to minimize copy/paste risk.

Reply:
Done